crates/matrix-sdk/src/widget_api/messages/from_widget/message_types.rs

```diff
@@ -0,0 +1,25 @@
+use serde::{Deserialize, Serialize};
+
+use url::Url;
+
+use crate::widget_api::messages::message::MessageBody;
+
+#[derive(Serialize, Deserialize, Debug, Clone)]
+#[serde(tag = "action")]
+pub enum FromWidgetMessage {
+    #[serde(rename = "supported_api_versions")]
+    GetSupportedApiVersion(MessageBody<(), SupportedVersions>),
```

Comment:
NB: `Request` was exchanged for `MessageBody`, yet they have different semantics.

(this probably was not intended in the rename PR?)

Reply:
changed it back to request